analytics/src/test/java/org/georchestra/analytics/StatisticsControllerIT.java

```diff
@@ -0,0 +1,84 @@
+package org.georchestra.analytics;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.dbcp.BasicDataSource;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@WebAppConfiguration
+@ContextConfiguration(locations = { "classpath:/ws-servlet.xml" })
+public class StatisticsControllerIT {
+
+    private @Autowired StatisticsController controller;
+
+    private @Value("${dlJdbcUrlOGC}") String dlJdbcUrlOGC;
+
+    private @Autowired BasicDataSource ds;
+
+    protected final Log logger = LogFactory.getLog(getClass().getPackage().getName());
+
+    private static boolean dataLoaded = false;
+
+    public @Before void before() throws Exception {
+        if (!dataLoaded) {
```

Comment:
I think @groldan provided a better approach while working on GPDR:

https://github.com/georchestra/georchestra/pull/2613/files#diff-0cdf168c2c66425c514a83589eeddd9dR62

The annotation comes from com.github.database.rider and should do the work better than I can do in this before() method

Reply:
IIUC, you do not consider this one is ready for merge ?

Reply:
I was wondering if it was worth to integrate this database-rider dependency (on a test scope) or not. In the current state, it works, it would be fancier with it.

Reply:
I did some rapid tests with database-rider (the `DataSet` annotation), as well as the `Sql` from spring-tests, but with no luck, I cannot get them work to load my fixtures.

Reply:
trying harder I succeeded \o/ I'll update my PR actually